### PR TITLE
fix: 26 fix reformat references section 3/12

### LINKED ISSUE #26: fix: Reformat References section 3/12

### PR DESCRIPTION
**Pull request by musslick**
Resolves #26

### CODE REVIEW

**Review by listar2000 — COMMENTED** [review]
Maybe you need to add `^` before the 40 & 41 links as well?

File: git-text-content.md [hunk 1]
```diff
@@ -301,8 +301,8 @@ References[edit]
 ^ "Subversion Jobs, Average Salary for Apache Subversion (SVN) Skills". Itjobswatch.co.uk. Archived from the original on 25 October 2016. Retrieved 30 September 2016.
 ^ "Mercurial Jobs, Average Salary for Mercurial Skills". Itjobswatch.co.uk. Archived from the original on 23 September 2016. Retrieved 30 September 2016.
 ^ "VSS/SourceSafe Jobs, Average Salary for Microsoft Visual SourceSafe (VSS) Skills". Itjobswatch.co.uk. Archived from the original on 29 October 2016. Retrieved 30 September 2016.
-^ "Windows switch to Git almost complete: 8,500 commits and 1,760 builds each day". Ars Technica. 24 May 2017. Archived from the original on 24 May 2017. Retrieved 24 May 2017.
-^ "git-init". Git. Archived from the original on 15 March 2022.
+[^38]: "Windows switch to Git almost complete: 8,500 commits and 1,760 builds each day". Ars Technica. 24 May 2017. Archived from the original on 24 May 2017. Retrieved 24 May 2017.
+[^39]: "git-init". Git. Archived from the original on 15 March 2022.
```
Comment (hollandjg) [inline git-text-content.md:305]: This looks like it's bleeding over into the next reference.
```suggestion
[^39]: "git-init". Git. Archived from the original on 15 March 2022.

```

**Review by hollandjg — CHANGES_REQUESTED** [review]
This looks great, thanks. There's one suggestion I have.

**Review by hollandjg — APPROVED** [review]
This looks great!

**Pull request merged**